aptos-move/framework/aptos-framework/sources/aggregator_v2/aggregator_v2.move

```diff
@@ -1,6 +1,16 @@
 /// This module provides an interface for aggregators (version 2).
 module aptos_framework::aggregator_v2 {
-    use std::option::{Self, Option};
+    use std::error;
+
+    /// The value of aggregator overflows. Raised by uncoditional add() call
+    const EAGGREGATOR_OVERFLOW: u64 = 1;
+
+    /// The value of aggregator underflows (goes below zero). Raised by uncoditional sub call
+    const EAGGREGATOR_UNDERFLOW: u64 = 2;
+
+    /// Tried casting into a narrower type (i.e. u64), but aggregator range of valid values
+    /// cannot fit (i.e. limit exceeds type::MAX)
+    const EAGGREGATOR_LIMIT_ABOVE_CAST_MAX: u64 = 2;
```

Comment:
Maybe add a comment that this is raised by native code?

Reply:
I'm curious - what do people see as the external error if this happens? Do they get this nice error or something more opaque?